pkg/controllers/status/work_status_controller.go

```diff
@@ -231,7 +231,6 @@ func (c *WorkStatusController) syncWorkStatus(key util.QueueKey) error {
 		return err
 	}
 
-	util.MergeLabel(desiredObj, workv1alpha2.WorkPermanentIDLabel, util.GetLabelValue(workObject.Labels, workv1alpha2.WorkPermanentIDLabel))
```

Comment:
Here can it be removed?

When karmada has an `agent`, it should be the `agent` that adds these labels, right?

Reply:
we add it in work mutating-webhook now, so it can be safely removed.

Reply:
What I mean is, this was originally handled by the karmada-agent, but it has been moved to be managed by the control panel. I'm not sure if this is appropriate.